external/anomaly/ote_anomalib/__init__.py

```diff
@@ -15,3 +15,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions
 # and limitations under the License.
+
+from .task import BaseAnomalyTask
+from .openvino import OpenVINOAnomalyTask
+
+__all__ = ["BaseAnomalyTask", "OpenVINOAnomalyTask"]
```

Comment:
Is `OpenVINOAnomalyTask` also base?

Reply:
I named it base task to distinguish it from the OpenVINO task. So we'll have the Base task and the OpenVINO task. Alternatively, we could rename it to `AnomalyTask`